app/service/app_svc.py

```diff
@@ -18,7 +19,7 @@
 from app.service.interfaces.i_app_svc import AppServiceInterface
 from app.utility.base_service import BaseService
 
-Error = namedtuple('Error', ['name', 'msg', 'optional'])
+Error = namedtuple('Error', ['name', 'msg'])
```

Comment:
Any reason why the optional flag was removed here? That's used by the UI in v4 to determine whether or not a dependency is required or not in the startup messages.

Reply:
Honestly I have no idea why that changed, I looked in the history and it looks a change you made when you commited to the Magma-cors branch with my intial changes, in this commit:

https://github.com/mitre/caldera/commit/5fe04ade03705067cdeb12ba0d65426a2f5f0be0

Reply:
![image](https://user-images.githubusercontent.com/6034483/176733506-8e8d403a-358b-427f-a951-49879a5e9b9a.png)

It was me all along